osu.Game.Rulesets.Mania/Edit/ManiaHitObjectComposer.cs

```diff
@@ -119,5 +119,8 @@ protected override void UpdateAfterChildren()
                     beatSnapGrid.SelectionTimeRange = null;
             }
         }
+
+        public override IEnumerable<string> ConvertSelectionToString()
```

Comment:
This should return a `string`, not an `IEnumerable<string>`. You aren't using the enumerable part and it also doesn't match the method name.